quest_tool_plugins/raster/rst_base.py

```diff
@@ -1,26 +1,24 @@
-"""Functions required run raster filters"""
-
 import rasterio
 
-from quest.plugins import ToolBase
 from quest import util
+from quest.plugins import ToolBase
 from quest.api import get_metadata
-
+from quest.static import DataType, UriType
 
 
 class RstBase(ToolBase):
     # metadata attributes
-    group = 'raster'
-    operates_on_datatype = ['raster']
+    group = DataType.RASTER
```

Comment:
```suggestion
    group = 'raster'
```

Eventually we need to make these use the `static.FilterGroup` enum once that is implemented.